test/unit/org/apache/cassandra/Util.java

```diff
@@ -1007,4 +1008,38 @@ public static File relativizePath(File targetBasePath, File path, int components
         return new File(targetBasePath.toPath().resolve(relative));
     }
 
+    public static void flush(ColumnFamilyStore cfs)
+    {
+        Util.flush(cfs);
+    }
```

Comment:
Infinite recursion.

Reply:
That's what happens when I "replace in files" with a regex.... Sorted now.